src/control/SConscript

```diff
@@ -108,7 +108,7 @@ def scons():
 
     denv.d_setup_go()
 
-    prereqs.require(denv, 'ofi', 'hwloc')
```

Comment:
We do use hwloc in the control plane, just FYI. If this works though then I don't have a problem with it.